orderbook/src/database/orders.rs

```diff
@@ -56,6 +56,23 @@ impl DbOrderKind {
     }
 }
 
+#[derive(sqlx::Type)]
+#[sqlx(type_name = "BalanceFrom")]
+#[sqlx(rename_all = "snake_case")]
+pub enum BalanceFrom {
```

Comment:
Nit: maybe docs on what the subtypes mean (especially owner vs VaultExternal might not be obvious from the name)

Reply:
All fixed up with some very nice documentation. Thanks very much for pointing this out!